chapter_1/exercise_1_09/copy_io.c

```diff
@@ -2,8 +2,15 @@
 
 int main(void)
 {
-  char c;
-  char last_c = '\0';
+  int c;
+  int last_c = '\0';
+  /*
+    If I use chars here instead of ints I get a "comparison
+    between pointer and integer gcc" warning.
+    getchar returns int not char.
+    Char type is de facto a 1-byte integer type.
+    Thank you for you solutions, they're helping me learn.
+  */
```

Comment:
Please remove your comment here, and we can merge your fix.

Reply:
Done. Thank you :)